pkg/chunk/encoding/doubledelta.go

```diff
@@ -133,26 +139,28 @@ func (c doubleDeltaEncodedChunk) Add(s model.SamplePair) ([]Chunk, error) {
 		}
 	}
 	if tb != ntb || vb != nvb || c.isInt() != nInt {
-		if len(c)*2 < cap(c) {
-			return transcodeAndAdd(newDoubleDeltaEncodedChunk(ntb, nvb, nInt, cap(c)), &c, s)
-		}
+		// TODO(codesome): How to take care of this case?
+		// if len(c.b)*2 < cap(c.b) {
+		// 	return transcodeAndAdd(newDoubleDeltaEncodedChunk(ntb, nvb, nInt, cap(c.b)), &c, s)
```

Comment:
I have commented out all the `transcodeAndAdd` for now as they return a slice of chunks. Given they are called within the same chunk type, can it be assumed that the single destination chunk can hold the existing chunk data and the sample (in short a slice of only 1 chunk, which looks like the case wherever it is called)?